nixos/doc/manual/installation/installing.xml

```diff
@@ -6,9 +6,18 @@
 
 <title>Installing NixOS</title>
 
+<para>NixOS can be installed on BIOS or UEFI systems.  The procedure
+for a UEFI installation is by and large the same as a BIOS installation. The differences are mentioned in the steps that follow.</para>
+
 <orderedlist>
 
-  <listitem><para>Boot from the CD.</para></listitem>
+  <listitem><para>Boot from the CD.</para>
+    <variablelist>
+      <varlistentry><term>UEFI systems</term>
+      <listitem><para>You should boot the live CD in UEFI mode
+  (consult your specific hardware's documentation for instructions).
```

Comment:
should we be recommending the refind tool? What benefit does that give the user?

Reply:
At this point I'm just copying the existing information from installing-uefi.xml into this document :)

Reply:
iirc, refind is the only way to do it on mac?

Reply:
That is the kind of thing that motivated me to try to change as little as possible while putting this together :)

Reply:
On my old laptop (2011 macbook pro) I just used systemd-boot, no refind involved.

Reply:
In this PR I'm trying to merge existing documentation and add some information about fdisk - I don't want to involve more than that or the review will go on forever and/or I'd be writing about things I have no first hand experience with.

Maybe we can file a docs issue about refind?